src/ci/resources/papi_v2_application.conf

```diff
@@ -97,5 +97,16 @@ backend {
         }
       }
     }
+    papi-v2-usa {
+      actor-factory = "cromwell.backend.google.pipelines.v2alpha1.PipelinesApiLifecycleActorFactory"
+      config {
+        include "papi_provider_config.inc.conf"
+        filesystems.http {}
+        filesystems.gcs.auth = "user_service_account"
+        filesystems.drs.auth = "user_service_account"
+        project = "user_error: google_project must be set in workflow options http://cromwell.readthedocs.io/en/develop/wf_options/Google/"
```

Comment:
I don't understand this being in the config file...?

Reply:
This is [copypasta from CaaS](https://github.com/broadinstitute/cromwell-develop/blob/f2abca1a0ea20087204ded89959789c4dfde8eea/configs/caas/cromwell.conf.ctmpl#L387).

AFAIK we haven't coded have a way of removing config values AND including a user friendly error message. When the USA workflow option support was tacked onto the other auth modes, we left in the code paths that still look for `project` in the backend config.

Using this config "value" is the closest to providing helpful messages. It leads to errors in the returned failure metadata such as (paraphrasing):

> java.lang.RuntimeException: Invalid project: user_error: google_project must be set in workflow options http://cromwell.readthedocs.io/en/develop/wf_options/Google/

versus the very unhelpful:

> java.lang.RuntimeException: Config key not found `project`